azurerm/internal/services/network/express_route_circuit_connection_resource.go

```diff
@@ -0,0 +1,272 @@
+package network
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-11-01/network"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
```

Comment:
This is wants to be also replaced by pluginsdk

Reply:
Updated